java/common/src/main/java/io/strimzi/common/ConfigUtil.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.common;
+
+public class ConfigUtil {
+
+    public static String convertEnvVarToPropertyKey(String envVar) {
+        return envVar.substring(envVar.indexOf("_") + 1).toLowerCase().replace("_", ".");
+    }
+
+}
```

Comment:
I would just add a little bit of Javadoc both on class and method to avoid Java 17 support complaining in the future.

Reply:
@ppatierno javadoc added to ConfigUtil class in Common module.